packages/mongodb-rag-ingest/src/commands/all.test.ts

```diff
@@ -142,4 +136,119 @@ describe("allCommand", () => {
       await ingestMetaStore.close();
     }
   });
+
+  it("deletes pages and embedded content that are no longer in the data sources", async () => {
+    const ingestMetaStore = makeIngestMetaStore({
+      connectionUri: uri,
+      databaseName,
+      entryId: "all",
+    });
+    assert(mongoClient);
+
+    // Mock data sources
+    const mockDataSources: DataSource[] = [
+      {
+        name: "source1",
+        fetchPages: async () => [
+          {
+            url: "test1.com",
+            format: "html",
+            sourceName: "source1",
+            body: "hello source 1",
+          },
+        ],
+      },
+      {
+        name: "source2",
+        fetchPages: async () => [
+          {
+            url: "test2.com",
+            format: "html",
+            sourceName: "source2",
+            body: "hello source 1",
+          },
+        ],
+      },
+    ];
+    const mockDataSourceNames = mockDataSources.map(
+      (dataSource) => dataSource.name
+    );
+
+    try {
+      // Start Setup
+      // create pages and verify that they have been created
+      await updatePages({ sources: mockDataSources, pageStore });
+      const pages = await pageStore.loadPages();
+      assert(pages.length == 2);
+      // create embeddings for the pages and verify that they have been created
+      await updateEmbeddedContent({
+        since: new Date(0),
+        embeddedContentStore: embedStore,
+        pageStore,
+        sourceNames: mockDataSourceNames,
+        embedder,
+      });
+      let page1Embedding = await embedStore.loadEmbeddedContent({
+        page: pages[0],
+      });
+      let page2Embedding = await embedStore.loadEmbeddedContent({
+        page: pages[1],
+      });
+      assert(page1Embedding.length);
+      assert(page2Embedding.length);
+      // End Setup
```

Comment:
can you refactor to include this set up stuff in a `beforeAll/Each` block? and the tear down in an `afterAll/Each`?

general practice is to include that type of stuff in the set up/tear down blocks rather than the test itself